### ISSUE TITLE
OCW Retro

### ISSUE THREAD

**Issue by bartr** [retaildevcrews/OCW#220]
# Retrospected Session

### Session details:

**Date**: Friday, 21 May 2021 at 16:09:37 GMT-05:00

**URL**: https://rdc-retro.azurewebsites.net/game/T4nBVQIj-

**Participants**: Bags, Hussain, Jerry, Johee, Sean, aakindele, art, bartr, jofultz, wendy

**Posts**: 45

**Votes**: 133

### Actions

- During challenges, encourage challenge teams to take breaks
- Ask if it's their first hack / OCW
- Can we delete the k8s deployment and keep the network?

#### Liked

- (+6/-0) Everyone actively participated
- (+5/-0) #PairingIsCaring
- (+5/-0) Everyone helped unblock each other
- (+4/-0) #SharingIsCaring
- (+3/-0) PnP ASB is a solid base to build on
- (+3/-0) OCW resulting in updates to ASB offering
- (+2/-0) Liked the sessions
- (+2/-0) Those long-running tasks in the setup script can actually run really fast so don't go for too long of a walk when you run them.
- (+2/-0) The whole shopping activity turned out to be more entertaining than I thought it would be. It was a fun way to learn about people and find some neat things.
- (+2/-0) Hackers felt comfortable to add to triage
- (+2/-0) LOVED the challenges. Felt good to contribute directly to the project
- (+1/-0) Positive feedback from PnP team about the hacks folks are working on
- (+1/-0) From time to time we hit a blocker that would have taken 30 minutes to work through but the coaches were ready and available most all the time to jump in and save us time through these things. That was nice to keep us moving.

  
#### Learned

- (+42/-0) Calf pulling 101
- (+6/-0) Shout out to AK for a great presentation and great coaching!  Thank you!
- (+4/-0) Shout out to the Austin Crews for building high quality content
- (+4/-0) AK Rocks
- (+3/-0) Keep issues in "one repo" instead of 5
- (+3/-0) Building an "open hack" is a LOT of work!
- (+3/-0) Shout out to Deanna for herding the ninja cats
- (+3/-0) Reinforcement that good docs focused on developer experience are very important
- (+2/-0) Mapping branches 1:1 to clusters works well and alleviates the need for multiple repos
- (+2/-0) Istio is easier to configure than Traefik
- (+2/-0) Why quality gates and testing are important in DevOps. Twice in as many weeks that someone has turned off an offering and caused churn. I'm looking at you AppGatewayAnalytics and don't you pretend like you didn't do anything yesterday Codespaces!
- (+2/-0) If Sharing is Caring and Pairing is Caring then it only follows that Sharing is Pairing. And yet... ;-)
- (+2/-0) Challenges really helped me learn more than I could have from just following the setup commands
- (+1/-0) Key Vaults don't actually delete. At least not right away.
- (+1/-0) GitHub Code Spaces is amazing. A real time saver and a cool way to get up and going fast.
- (+1/-0) Learned a lot from this upskilling session, this was really well organized and I feel confident working with AKS in the future,

  
#### Lacked

- (+3/-0) Would have been nice to have a cadence for breaks. Sometimes we went forever without one, and other times we took a break but when I got back to the meeting some people didn't leave and kept going causing a gap in the team following. It was not a crisis, but it could have been managed easier with, maybe, a kind of team agreement or something.
- (+2/-0) Lacked good end-to-end docs for Traefik configs and features.
- (+2/-0) Copy/Paste is not all it's cracked up to be.

  
#### Longed For

- (+2/-0) In person One CSE Week :(
- (+2/-0) Bash Jupyter Notebook
- (+2/-0) Enjoyed hearing about some of the team having the chance to have happy hour together. Sounded fun. I just sat on my couch, of course. But, I'm glad you guys had a nice time.
- (+2/-0) Faster way to test

  
#### Word Cloud: Words to describe this week

- (+6/-0) Learning
- (+5/-0) Fun
- (+5/-0) Pairing
- (+5/-0) Phone-a-Friend
- (+5/-0) Collaboration
- (+4/-0) Team
- (+4/-0) Engaging
- (+4/-0) SuperFunWorkingWithTeam
- (+2/-0) Sharing
